webapp/components/NavBar/NavBar.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div>
+    <template v-if="loggedIn">
+      <button id="logoutBtn" @click="logout">Logout</button>
+    </template>
+    <template v-else>
+      <NuxtLink to="/login">Login</NuxtLink>
+    </template>
+  </div>
```

Comment:
```suggestion
  <div>
    <button v-if="loggedIn" id="logoutBtn" @click="logout">Logout</button>
    <NuxtLink v-else to="/login">Login</NuxtLink>
  </div>
```


You could even remove the `<div>` if you want. An if-clause with an else is allowed as root node.